src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,99 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int ARRAY_SIZE = 10;
+    private static final int FIRST_INDEX = 0;
+    private static final int INCREMENT = 1;
+    private static final double COEFFICIENT_INCREASE_ARRAY = 1.5;
+    private T[] array;
+    private int size;
+
+    public ArrayList() {
+        array = (T[]) new Object[ARRAY_SIZE];
+    }
+
+    private void acceptIndex(int index) {
```

Comment:
all private methods should be after public ones

Reply:
done